components/LogosMarquee/Slider.tsx

```diff
@@ -47,7 +47,7 @@ export const ProviderMarquee = memo(() => {
         scatterRandomly
         resetAfterTries={200}
       >
-        {Object.entries(manifest.providers)
+        {Object.entries(manifest.providersOAuth)
```

Comment:
Let's showcase the email providers too!
```suggestion
        {Object.entries({...manifest.providersOAuth, ...manifest.providersEmail})
```